ispyb/model/integration.py

```diff
@@ -85,3 +91,93 @@ def __str__(self):
     ('detector_distance', 'refinedDetectorDistance'),
     ('timestamp', 'recordTimeStamp'),
 ))
+
+class UnitCell():
+  '''An object representing the parameters of the unit cell I.e unit cell edges and angles
+  '''
+
+  def __init__(self, a,b,c,alpha,beta,gamma):
+    '''Unit cell object
+
+       :param apiid: AutoProcIntegrationID
+       :param db_conn: ISPyB database connection object
+       :return: A unit cell object representing the database entry for
+                the specified AutoProcIntegrationID
```

Comment:
Could you please update this comment to describe the 6 parameters using the same syntax (```:param X: what the parameter means```) and remove the reference the ```AutoProcIntegrationID``` reference in the ```:return:``` bit